### ISSUE TITLE
upload photos from webcam using a mobile device - stop working

### ISSUE THREAD

**Issue by josemvm** [3liz/lizmap-web-client#2427]
### What is the bug?

lizmap lost the ability to upload images via camera, the user can only choose browser files

### Steps to reproduce the issue

...

### Lizmap version

3.4.4

### QGIS desktop version

3.16.9

### QGIS server version

3.16.9

### Operating system

Android 10

### Browsers

Firefox

### Browsers version

90.1.3

**Comment by Gustry** [comment]
It works perfectly on https://demo.lizmap.com/lizmap/index.php/view/map/?repository=feat1&project=observatoire with my phone and Firefox 90.1.3.

**Comment by josemvm** [comment]
your lwc version?

**Comment by Gustry** [comment]
You can try all of them, it all works for the camera.

https://demo.lizmap.com/lizmap/index.php/view/map/?repository=feat1&project=observatoire => 3.5
https://demo.lizmap.com/lizmap_3_5/index.php/view/map/?repository=feat1&project=observatoire
https://demo.lizmap.com/lizmap_3_4/index.php/view/map/?repository=feat1&project=observatoire
https://demo.lizmap.com/lizmap_3_3/index.php/view/map/?repository=demo&project=observatoire

But actually, I have noticed, I can't choose any file on my phone. So if I understand, it's the reverse from you ?

**Comment by josemvm** [comment]
hi @Gustry thanks for your feedback

i can confirm with your observatoire projects on my phone with firefox.
i have also my own projects that ask for webcam and i have others that only ask for files and i don´t know why...

i' am trying with other browsers (opera, chrome, edge) and all of them ask for webcam, audio or files but not with firefox

**Comment by laurentj** [comment]
The enabling of the webcam capture depends on many parameters.
The first one is [the `capture` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture). In lizmap, we set it to `camera` but it seems this is not any more a good value. HTML specifications have changed many times since few years. In my patch #2617 , I set it to `environment`.

However, the `capture` attribute is not supported by all browsers. It seems Firefox does not support it but perhaps [this compatibility matrix](https://caniuse.com/html-media-capture) is not updated.

The second parameter seems to be the type of file we indicate into the `accept` attribute, so into QGIS. It should be only an image type. Perhaps the type is not set correctly into your projects.
